assets/agenda/components/AgendaCoverages.jsx

```diff
@@ -61,7 +61,7 @@ function AgendaCoveragesComponent({item, coverages, wireItems, actions, user, on
                     >
                         <span className={classNames('coverage-item__coverage-icon', WORKFLOW_COLORS[coverage.workflow_status])}>
                             <i className={`icon-small--coverage-${getCoverageIcon(coverage.coverage_type)} me-2`}></i>
-                            <span>{`${getCoverageDisplayName(coverage.coverage_type)}${getSlugline(coverage)}`}</span>
+                            <span>{`${coverage.genre.length !==0 ? gettext(coverage.genre[0].name) : getCoverageDisplayName(coverage.coverage_type)}${getSlugline(coverage)}`}</span>
```

Comment:
In this context, Genre is not the Coverage Type. Coverage type is the type of content that is being planed for (https://github.com/superdesk/newsroom-core/blob/develop/newsroom/web/default_settings.py#L281)

Reply:
but we have to show that genre name instead of type, IMO that is ticket AC. and with these changes it looks like this 
<img width="1512" alt="Screenshot 2023-06-22 at 4 43 52 PM" src="https://github.com/superdesk/newsroom-core/assets/73937490/61207778-78f5-4462-b9f3-bd881a1736e7">

Reply:
Your changes will break the system for other customers.
The genre should be shown, as well as the content type.

Reply:
From the ticket:
```I want to see what type of text has been planned (genre)```

The Text is the Content Type, and Genre explains what type of text would be expected

Reply:
so you mean instead of removing the functionality of display content_type, we add genre also there right ?

Reply:
> Your changes will break the system for other customers. The genre should be shown, as well as the content type.

I see, I'm sorry, because before I created this PR, I confirmed with Gideon about the changes and shared the screenshots with it.

Reply:
Hi, @MarkLark86 ,can we do something like this ?
![image](https://github.com/superdesk/newsroom-core/assets/73937490/c580d63f-42e2-4969-9305-39d4c2297ec1)